projects/Heads/module.nix

```diff
@@ -0,0 +1,22 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.programs.heads;
+in
+{
+  # Note: Heads produces ROM images intended to be flashed onto real hardware.
+  # This module only exists to allow automated testing of a QEMU-targeting
+  # Heads ROM.
+  options.programs.heads = {
+    enable = lib.mkEnableOption "symlinking the Heads ROM for qemu-coreboot-fbwhiptail-tpm1-hotp";
+  };
```

Comment:
We do have binary artefacts in our data model now!

Reply:
This is used in the test, to have a fixed location where we can find the ROM at. @eljamm said that currently, it's not possible to refer to the `binary` attrset.

Also, referencing `pkgs` in `binary` (in order to point it at the artifact from the build), currently makes eval blow up, which is why it's commented out: https://github.com/ngi-nix/ngipkgs/pull/773

Reply:
> @eljamm said that currently, it's not possible to refer to the binary attrset.

Because it's [not mapped back](https://github.com/ngi-nix/ngipkgs/blob/51793ff4b032d718e21a5f15620171126f5a26f8/default.nix#L110-L119), but we can do this if we need to.